src/routes/global-components/TableOfContent.svelte

```
@@ -1,28 +1,31 @@
 <script lang="ts">
 	import { Link } from '$lib';
```

ESLint

[error] 'Link' is defined but never used.

(@typescript-eslint/no-unused-vars)